Ginger/Ginger/MainWindow.xaml.cs

```diff
@@ -1016,6 +1026,165 @@ void HideSplash()
                 });
         }
 
-        
+        public void AddHelpLayoutToShow(string helpLayoutKey, FrameworkElement focusedControl, string helpText)
+        {
+            if (WorkSpace.Instance.RunningInExecutionMode)
+            {
+                return;//not showing help in automatic run mode
+            }
+            HelpLayoutArgs helpLayoutArgs = new HelpLayoutArgs(helpLayoutKey, focusedControl, helpText);
+            if (WorkSpace.Instance.UserProfile.ShownHelpLayoutsKeys.Contains(helpLayoutArgs.HelpLayoutKey) == false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4555652)